src/redux/books/books.js

```diff
@@ -0,0 +1,40 @@
+const BOOK_ADDED = 'BOOK_ADDED';
+const BOOK_REMOVED = 'BOOK_REMOVED';
```

Comment:
- [ ] As it is a requirement for this project, the value of each action type needs to follow the [ducks pattern](needSupport), please kindly fix that.